examples/gbasf2_fei_training/fei_grid_workflow.py

```diff
@@ -0,0 +1,473 @@
+    stage = luigi.IntParameter()
+    mode = luigi.Parameter()
+
+    def output(self):
+
+        for outname in fei_analysis_outputs[self.stage]:
+            yield self.add_to_output(outname)
+
+    def requires(self):
+
+        if self.stage == -1:
+
+            return []  # default implementation, as in the luigi.Task (if no requirements are present)
+
+        else:
+
+            # need the timestamp of the additional inputs to build their TMP-SE path
+            yield PrepareInputsTask(
+                mode="AnalysisInput",
+                stage=self.stage - 1,
+                remote_tmp_directory=luigi.get_setting("remote_tmp_directory"),
+                remote_initial_se=luigi.get_setting("remote_initial_se"),
+            )
+
+            # need a symlink to the merged mcParticlesCount.root file
+            yield MergeOutputsTask(
+                mode="Merging",
+                stage=-1,
+                ncpus=luigi.get_setting("local_cpus"),
+            )
+
+            # need symlinks to *.xml files of FEI training of previous stages
+            for fei_stage in range(self.stage):
+
+                yield FEITrainingTask(
+                    mode="Training",
+                    stage=fei_stage,
+                )
+
+    def create_path(self):
+
+        luigi.set_setting("gbasf2_cputime", grid_cpu_time[self.stage])
+
+        # determine the remote TMP-SE destination of input tarball for gbasf2 command
+        if self.stage > -1:
+            timestamp = open(f"{self.get_input_file_names('successfull_input_upload.txt')[0]}", "r").read().strip()
+            additional_file = os.path.join(luigi.get_setting("remote_tmp_directory").rstrip('/')+timestamp,
+                                           "stage"+str(self.stage - 1), "sub00", "fei_analysis_inputs.tar.gz")
+            luigi.set_setting("gbasf2_input_datafiles", [additional_file])
+
+            # create symlinks to files, which are needed for current FEI analysis stage
+            for key in self.get_input_file_names():
+                if key.endswith(".xml"):
+                    filepath = '"' + self.get_input_file_names(key)[0] + '"'
+                    adjusted_key = '"' + key + '"'
+                    os.system(f"ln -sf {filepath} {adjusted_key}")
+            # need extra line for mcParticlesCount.root symlink, since removed otherwise
+            os.system(f"ln -sf {self.get_input_file_names('mcParticlesCount.root')[0]} mcParticlesCount.root")
+
+        path = create_fei_path(filelist=[], cache=self.cache, monitor=self.monitor)
+
+        if self.stage > -1:
+            # remove symlinks and not needed Summary.pickle files
+            for key in self.get_input_file_names():
+                if key == "mcParticlesCount.root" or key.endswith(".xml"):
+                    adjusted_key = '"' + key + '"'
+                    os.system(f"rm {adjusted_key}")
+            os.system("rm -f Summary.pickle*")
```

Comment:
use `os.remove` (see same suggestion in #91)

Reply:
Changed.